src/AzureStack/AzureStack.psd1

```diff
@@ -127,11 +127,7 @@
             # IconUri = ''
 
             # ReleaseNotes of this module
-            ReleaseNotes = '* AzureStack Operator module version 2.1.0. Requires AzureStack Hub Update 2102+,
-            * Compatible with Az module version 1.10.0
-            * The indvidual module versions are updated to 1.0.0, marking GA
-            * Support for changing the default Gpu partition on a GPU enabled Azure Stack hub with Set-AzsScaleUnit -NumberOfGPUPartition X
-            * Please refer to https://aka.ms/azspsh210 for additional instructions'
+            ReleaseNotes = '* Add update files to sign.'
```

Comment:
'* Bug fix - Adding signing for a file under utils folder in each of the module'